controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -437,6 +465,7 @@ func (r *StorageConsumerReconciler) reconcileCephRadosNamespace(
 	radosNamespaceName string,
 	additionalOwner client.Object,
 	builtinBlockPools []string,
+	forceDelete bool,
```

Comment:
Params should describe desired state, as such the nam of the param should be something like: 
```suggestion
	hasForceDeleteAnnotaion bool,
```